docs/lambda/configure-lambda-widget.asciidoc

```diff
@@ -0,0 +1,99 @@
+++++
```

Comment:
@trentm @bmorelli25 First -- this widget looks super cool.  I didn't realize this sort of thing was possible with ASCII Doc.  💯 

Second -- A couple of questions about the widget (apologies if these are retreads of discussions had elsewhere) -- 

1. Is this using an authenticated or unauthenticated request to `api.github.com`?  It looks like the later to me -- if so, do we have any concerns about hitting GitHub's API limits for public requests? Per [the docs](https://docs.github.com/en/rest/overview/resources-in-the-rest-api) -- _For unauthenticated requests, the rate limit allows for up to 60 requests per hour. Unauthenticated requests are associated with the originating IP address, and not the person making requests.?_

2. If there anything special we need to do/maintain in order to make sure this continues to work with future releases? (It looks like all we need to do is make sure the full ARNs are posted to the releases tab in GitHub as part of the release body text -- is there more to it than that?)

Reply:
>  I didn't realize this sort of thing was possible with ASCII Doc.

All credit to @AlexanderWert and Brandon.

> do we have any concerns about hitting GitHub's API limits for public requests?

It'll be limited by the IP of the personal reading the docs... so not *super* concerned. However, yah, I think the templating could perhaps improve to have reasonable content for if/when that request fails. Currently it'll look a little confusing. I opened https://github.com/elastic/apm-aws-lambda/issues/171 for this.

> anything special we need to do/maintain in order to make sure this continues to work with future releases?

My understanding is just that: full ARNs in the releases content.